orchestrator/cli/domain_gen_helpers/resource_type_helpers.py

```diff
@@ -266,6 +266,10 @@ def _find_new_relations(block_name: str, relations: dict[str, set[str]]) -> set[
     return set(flatten((list(v) for k, v in relations.items() if block_name in k)))
 
 
+def _find_new_block_relations(block_name: str, relations: dict[str, list[BlockRelationDict]]) -> set[str]:
+    return set(flatten([r["name"] for r in v] for k, v in relations.items() if block_name in k))
```

Comment:
Why `block_name in k`?

Reply:
used the old logic above this fn, but you are right, no reason for this and `relations.get(block_name, set())` can be used instead